shared/triage_metadata.go

```diff
@@ -246,6 +264,17 @@ func appendTestName(test string, metadata MetadataResults) {
 	}
 }
 
+func containsInterop(metadata MetadataResults) bool {
+	for _, links := range metadata {
+		for _, link := range links {
+			if strings.Contains(link.Label, "interop") {
```

Comment:
Just to call out the behavior, this will catch labels like `interop-2023-has` but also `lets-go-interoperability` that contain "interop" anywhere. That's OK I think.

Reply:
I am happy to tighten the logic in a follow-up PR!